fs/tmpfs/fs_tmpfs.c

```diff
@@ -1605,7 +1605,15 @@ static ssize_t tmpfs_write(FAR struct file *filep, FAR const char *buffer,
 
   /* Handle attempts to write beyond the end of the file */
 
-  startpos = filep->f_pos;
+  if ((filep->f_oflags & O_APPEND) == O_APPEND)
```

Comment:
```suggestion
  if ((filep->f_oflags & O_APPEND) != 0)
```

Reply:
done

Reply:
Thanks, check 0 is faster than check constant